### ISSUE TITLE
[Bug]: Can't scan 6 or more videos when scanning a channel

### ISSUE THREAD

**Issue by Close13** [ThioJoe/YT-Spammer-Purge#363]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

When I tried to scan the most recent videos on a channel and chose to scan more than 5 videos, an error message displayed. 

### Release version

v2.10.0-Beta1

### Steps to reproduce

1. Choose to scan a channel
2. Choose a number greater than 5
3. Error message is shown

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
Number of Recent Videos: 6
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 4098, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 4080, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 3327, in primaryInstance
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 1137, in get_recent_videos
TypeError: 'NoneType' object is not subscriptable
------------------------------------------------
Error Message: 'NoneType' object is not subscriptable

Unknown Error - Code: Z-3 occurred.
```

**Comment by Firecul** [comment]
Tbf it does say that a max of 5 can be scanned. It could be a request to increase that number though.

**Comment by Close13** [comment]
Wasn't the limit increased to 500 in 2.9.0?

**Comment by Firecul** [comment]
Oh yeah, looks like you are right. I was on an old version, v2.9.0 works as expected atm. I'll get the newest and try that.

**Comment by compsup** [comment]
Problem is the `nextPageToken[0]`. 
Explanation: https://itsmycode.com/python-typeerror-nonetype-object-is-not-subscriptable/
![image](https://user-images.githubusercontent.com/70211072/148708507-6ac05ccb-82c0-4688-86ea-d818a637b9f7.png)

**Comment by ThioJoe** [comment]
Yea I screwed it up when adding the return-to-menu feature. That function was a bit tricky.
Should be fixed with be73e47e854e5e02b9fd12610b059f80e3137c93 and the latest 2.10.0-Beta2 Release
